GithubPractice/ViewController.swift

```diff
@@ -29,4 +36,58 @@ class ViewController: UIViewController {
     @IBAction func didClickClear(_ sender: Any) {
         
     }
+    
+    
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+    }
+    
+    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+        self.view.endEditing(true)
+    }
+    
+    
+    func textFieldShouldReturn(_ textField: UITextField) -> Bool {
+        textField.resignFirstResponder()
+        return true
+    }
+    
+    
+}
+
+@IBDesignable
+class HighlightButton: UIButton {
+    var highlightDuration: TimeInterval = 0.25
```

Comment:
A good practice is to move all constants in a separate global structure/ case-less enum